#!/usr/bin/env python
PACKAGE = "wm_admitance"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enableAdmitance", bool_t, 0, "Enable/disable admitance mode", False)
gen.add("verboseMode", bool_t, 0, "Enable/disable verbose mode", False)

exit(gen.generate(PACKAGE, "wm_admitance", "Admitance"))
